vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSpaceController.java

```diff
@@ -53,11 +53,16 @@
     public String space(@PathVariable("id") String id, Model model) {
         ISpace space = spaceManager.getSpace(id);
         List<ISpaceLinkDisplay> spaceLinks;
+        Boolean isSpacePublished = true;
         /* (non-Javadoc)
          * Below null check is added to accommodate already existing spaces with null space status
          */
         if (space.getSpaceStatus() == null || space.getSpaceStatus().equals(SpaceStatus.PUBLISHED)
                 || authenticationFacade.getAuthenticatedUser() != null) {
+            if(space.getSpaceStatus() == null || space.getSpaceStatus().equals(SpaceStatus.UNPUBLISHED)) {
```

Comment:
If a space has null as status, it's considered published. This should only be the case for spaces created before we added the status field and all of them are considered published.